### PR TITLE
Fix/language

### PR DESCRIPTION
**Pull request by jduss4**
Adds full language names to toggle and fixes some exciting stuff along the way!

image

closes https://github.com/CDRH/orchid/issues/206
closes https://github.com/CDRH/orchid/issues/154   (goes with Greg's suggestion, not Jess's bad idea)

### CODE REVIEW

File: docs/languages.md [hunk 1]
```diff
@@ -53,11 +60,50 @@ the app can't find a translation of the string.
...
+`&langs` definition, and make it available to your language's locales configuration.
+
+For example, if you were adding Japanese, your
+`config/locales/languages.yml` file would be:
+
+```yaml
+langs: &langs
+  en: English
+  es: Español
+  ja: 日本語
```
Comment (techgique) [inline docs/languages.md:83]: I think it's more common to abbreviate Japan(ese) as `jp`. `ja` might be Java(nese)

**Review by techgique — CHANGES_REQUESTED** [review]
One tiny change haha. I like this reorganization and the documentation updates :raised_hands:

File: docs/languages.md (hunk 1 above)
Comment (jduss4) [reply docs/languages.md:83]: Oh interesting.  I was using the LoC list here, although the abbreviation can be arbitrary so it's not like we need to stick to a particular authority list or anything:  https://www.loc.gov/standards/iso639-2/php/code_list.php

Comment (techgique) [reply docs/languages.md:83]: Oh okay. Works for me

**Pull request merged**